### DISCUSSION TITLE
Turn on WiFi using a QR code

### DISCUSSION THREAD

**Discussion by Anton778** [gopro/labs#1167] [Q&A]
Hello. I need to transfer my files from the camera to the computer using WIFI. However, the Hero 11 camera does not have a Wi-Fi power button. 
I installed the GoPro Labs firmware, there is an oW1 command using a QR code to turn on Wi-Fi. But in fact it does not turn on. Why is this happening?

**Comment by dnewman-gpsw** [comment]
I just tested, oW1 is not working on HERO11, but it is on HERO10, 12 and 13. Weird.  Hopefully there can be Labs release for HERO11 early in the new year.  I have filed the bug report.
